### PR TITLE
Monlog log broke due to broken code..

### PR DESCRIPTION
**Pull request by patku564**
An issue with log_malformed_data should now be corrected. Username -> username.
Since change of datetime import, datetime.datetime.now() became datetime.now().

Please verify and accept patch

### CODE REVIEW

**Comment by trew** [comment]
Holy wow. Maybe we shouldn't merge every branch into eachother as much as we do. I mean, this pull request doesn't really have changes from monlog_log only... It has the filter changes I did yesterday and those isn't done yet.

File: monlog/log/validation.py [hunk 1]
```diff
@@ -6,15 +6,27 @@
...
+        get_dict = bundle.request.GET
+        desc = data        
+        desc.update(client_ip)
+        desc.update(get_dict)       
+        desc = json.dumps(desc)
+        current_date = datetime.now()
+        monlog_user = User.objects.get(username="monlog")
+        severity = 4;
+
+        #return # TEMP, THIS FUNCTION IS UPDATED IN BRANCH MONLOG_LOG, BUT IS CURRENTLY CAUSING ME A LOT OF PROBLEMS! /TREW
```
Comment (trew) [inline monlog/log/validation.py]: This line. How do I remove it?

Comment (patku564) [inline monlog/log/validation.py]: I think i made a merge with the other branches. Which might be a bad idea. Maybe we should stick with merging only with master?
What do you mean "How do I remove it?" Doesn't it work after just deleting it? or what?

Comment (trew) [inline monlog/log/validation.py]: I was just thinking if I could make changes to the commits directly in github, but apparently I need to check out the code and make the changes.

**Pull request merged**